sycl-jit/jit-compiler/lib/rtc/DeviceCompilation.cpp

```diff
@@ -245,8 +258,14 @@ class SYCLToolchain {
     return TI.run();
   }
 
+  std::string_view getClangXXExe() const { return ClangXXExe; }
+
 private:
   clang::IgnoringDiagConsumer IgnoreDiag;
+  std::string ClangXXExe =
+      (jit_compiler::ToolchainPrefix + "/bin/clang++").str();
+  llvm::IntrusiveRefCntPtr<llvm::vfs::InMemoryFileSystem> ToolchainFS =
+      llvm::makeIntrusiveRefCnt<llvm::vfs::InMemoryFileSystem>();
```

Comment:
I believe we could have multiple threads calling `run(..)`; is this FS thread-safe to use then?